synapse/config/registration.py

```diff
@@ -128,6 +128,22 @@ def read_config(self, config, **kwargs):
             int
         ] = refreshable_access_token_lifetime
 
+        # The `nonrefreshable_access_token_lifetime` applies for tokens that can NOT be
+        # refreshed using a refresh token.
+        # If it is None, then these tokens last for the entire length of the session,
```

Comment:
Admins won't be able to input a literal None in the yaml; they'd have to use `null` or omit the key.

Reply:
Omitting the key seems in order and at least is consistent with `session_lifetime` which came before me.

Reply:
Oh sorry, I thought this comment would end up in the example yaml, where `None` wouldn't be meaningful. Ignore this.